cg/models/orders/excel_sample.py

```diff
@@ -29,6 +29,7 @@ class ExcelSample(OrderSample):
     concentration: Annotated[str, AfterValidator(numeric_value)] = Field(
         None, alias=ExcelSampleAliases.CONCENTRATION
     )
+    concentration_ng_ul: str = Field(None, alias=ExcelSampleAliases.CONCENTRATION_NG_UL)
```

Comment:
```suggestion
    concentration_ng_ul: str | None = Field(None, alias=ExcelSampleAliases.CONCENTRATION_NG_UL)
```

Reply:
And also, the field is marked as optional in the base class OrderSample, which also seems to handle this for cases where the user does not themselves give None as input.